tests/console/skopeo.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test skopeo installation and basic usage
+#    Cover the following aspects of skopeo:
+#      * Inspecting a repository before pulling it
+#      * copy an image from Docker registry into an OCI image-spec
+# Maintainer: Panagiotis Georgiadis <pgeorgiadis@suse.com>
+
+use strict;
+use base "consoletest";
+use testapi;
+use version_utils 'is_caasp';
+use caasp;
+
+sub run {
+    select_console("root-console");
+
+    record_info 'Test #1', 'Test: Installation';
+    my $package = 'skopeo';
+    if (is_caasp && check_var('FLAVOR', 'DVD') && check_var('SYSTEM_ROLE', 'plain')) {
+        script_run "rebootmgrctl set-strategy off";
+        trup_install($package);
```

Comment:
We don't have repositories on CaaSP 3.0 (systems are not registered for non-QAM tests) - trup_install won't find the skopeo package.

Reply:
Oh yes, it makes sense. In that case, how can I exclude CaaSP 3.0?

Reply:
If it will be included in 4.0 then you can use "unless is_caasp('3.0')". Otherwise is_caasp('qam').